similarity-search/src/main/java/org/phenotips/similarity/internal/SolrSimilarPatientsFinder.java

```diff
@@ -111,13 +115,19 @@ public void initialize() throws InitializationException
     @Override
     public List<PatientSimilarityView> findSimilarPatients(Patient referencePatient)
     {
-        return find(referencePatient, false);
+        return findSimilarPatients(referencePatient, null);
+    }
+
+    @Override
+    public List<PatientSimilarityView> findSimilarPatients(Patient referencePatient, String requiredConsentId)
```

Comment:
Why not the list of all possible (in future) consents? We have only one just for now.

Reply:
I thought about this, just no sure what is a good convenient way to do this (just alist may work, but is ugly on the calling side, when you have to create a list of one item all the time). We can upgrade to include a list later when we have a better idea about a use case

Reply:
varargs is easy.